dropwizard-validation/src/test/java/io/dropwizard/validation/ConstraintViolationsTest.java

```diff
@@ -31,64 +31,64 @@ public ConstraintPerson thePersonTest(@Valid ConstraintPerson person) {
         return person;
     }
 
-    @Test
-    public void invalidIntReturnIs500() {
-        Set<ConstraintViolation<ConstraintViolationsTest>> violations =
-                execValidator.validateReturnValue(
-                        this,
-                        getAccessibleMethod(getClass(), "theIntTest", Integer.class),
-                        4 // the return value
-                );
-
-        assertThat(violations).hasSize(1);
-        assertThat(ConstraintViolations.determineStatus(violations)).isEqualTo(500);
-    }
-
-    @Test
-    public void invalidIntParameterIs400() {
-        Set<ConstraintViolation<ConstraintViolationsTest>> violations =
-                execValidator.validateParameters(
-                        this,
-                        getAccessibleMethod(getClass(), "theIntTest", Integer.class),
-                        new Object[]{1} // the parameter value
-                );
-
-        assertThat(violations).hasSize(1);
-        assertThat(ConstraintViolations.determineStatus(violations)).isEqualTo(400);
-    }
-
-    @Test
-    public void invalidPersonReturnIs500() {
-        Set<ConstraintViolation<ConstraintViolationsTest>> violations =
-                execValidator.validateReturnValue(
-                        this,
-                        getAccessibleMethod(getClass(), "thePersonTest", ConstraintPerson.class),
-                        new ConstraintPerson() // return value has null name
-                );
-
-        assertThat(violations).hasSize(1);
-        assertThat(ConstraintViolations.determineStatus(violations)).isEqualTo(500);
-    }
-
-    @Test
-    public void invalidPersonParamIs400() {
-        Set<ConstraintViolation<ConstraintViolationsTest>> violations =
-                execValidator.validateParameters(
-                        this,
-                        getAccessibleMethod(getClass(), "thePersonTest", ConstraintPerson.class),
-                        new Object[]{new ConstraintPerson()} // the parameter value
-                );
-
-        assertThat(violations).hasSize(1);
-        assertThat(ConstraintViolations.determineStatus(violations)).isEqualTo(400);
-    }
-
-    @Test
-    public void invalidPersonRepresentationIs422() {
-        Set<ConstraintViolation<ConstraintPerson>> violations =
-                validator.validate(new ConstraintPerson());
-
-        assertThat(violations).hasSize(1);
-        assertThat(ConstraintViolations.determineStatus(violations)).isEqualTo(422);
-    }
+//    @Test
```

Comment:
Please don't just comment code but remove it completely. In the end that's what a version control system like git is for. :wink: